src/cascadia/TerminalApp/Pane.h

```diff
@@ -214,21 +214,24 @@ class Pane : public std::enable_shared_from_this<Pane>
     winrt::Windows::UI::Xaml::Controls::Grid _root{};
     winrt::Windows::UI::Xaml::Controls::Border _borderFirst{};
     winrt::Windows::UI::Xaml::Controls::Border _borderSecond{};
-    winrt::Microsoft::Terminal::Control::TermControl _control{ nullptr };
-    winrt::Microsoft::Terminal::TerminalConnection::ConnectionState _connectionState{ winrt::Microsoft::Terminal::TerminalConnection::ConnectionState::NotConnected };
     static winrt::Windows::UI::Xaml::Media::SolidColorBrush s_focusedBorderBrush;
     static winrt::Windows::UI::Xaml::Media::SolidColorBrush s_unfocusedBorderBrush;
 
+    // <--- SECTION FOR PROPERTIES THAT NEED TO BE TRANSFERRED BETWEEN CHILD/PARENT PANES UPON SPLITTING/CLOSING -->
     std::shared_ptr<Pane> _firstChild{ nullptr };
     std::shared_ptr<Pane> _secondChild{ nullptr };
     SplitState _splitState{ SplitState::None };
     float _desiredSplitPosition;
+    winrt::Microsoft::Terminal::Control::TermControl _control{ nullptr };
+    winrt::Microsoft::Terminal::TerminalConnection::ConnectionState _connectionState{ winrt::Microsoft::Terminal::TerminalConnection::ConnectionState::NotConnected };
+    winrt::Microsoft::Terminal::Settings::Model::Profile _profile{ nullptr };
+    bool _isDefTermSession{ false };
+    // <--- END SECTION -->

```

Comment:
```suggestion
#pragma region Properties that need to be transferred between child/parent panes upon splitting/closing
    std::shared_ptr<Pane> _firstChild{ nullptr };
    std::shared_ptr<Pane> _secondChild{ nullptr };
    SplitState _splitState{ SplitState::None };
    float _desiredSplitPosition;
    winrt::Microsoft::Terminal::Control::TermControl _control{ nullptr };
    winrt::Microsoft::Terminal::TerminalConnection::ConnectionState _connectionState{ winrt::Microsoft::Terminal::TerminalConnection::ConnectionState::NotConnected };
    winrt::Microsoft::Terminal::Settings::Model::Profile _profile{ nullptr };
    bool _isDefTermSession{ false };
#pragma endregion
```
nit: if you use `pragma region`, VS and VSCode let you expand/collapse it

Reply:
thanks!